[server] Enhance healthz handler with Kubernetes-compliant capability checks and Helm chart integration (#16457)
meshery/meshery: install/kubernetes/helm/meshery/templates/NOTES.txt

Meshery has been installed successfully!

> 1. Get the application URL by running these commands:

Reviewer: ```suggestion
1. Open Meshery UI by running commands from one of these options:
```

Meshery has been successfully deployed.

> Open Meshery UI using `mesheryctl` or `kubectl`:

Reviewer: ```suggestion
1. Open Meshery UI using `mesheryctl` or `kubectl`:
```

  echo "Visit http://127.0.0.1:9081/provider to use Meshery UI"
{{- end }}

2. Monitor deployment status:

>   kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "meshery.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -w

Reviewer: ```suggestion
2. Monitor deployment status.

- - - mesheryctl - - -

 mesheryctl system check

- - - OR kubectl - - - 

  kubectl get pods --namespace {{ .Release.Namespace }} -l "app.kubernetes.io/name={{ include "meshery.name" . }},app.kubernetes.io/instance={{ .Release.Name }}" -w
```
